docs/source/conf.py

```diff
@@ -178,8 +178,8 @@ def _transform_changelog(path_in: str, path_out: str) -> None:
 # documentation.
 
 html_theme_options = {
-    'pytorch_project': info.__homepage__,
-    'canonical_url': info.__homepage__,
+    'pytorch_project': 'https://pytorchlightning.ai',
+    'canonical_url': 'https://pytorch-lightning.readthedocs.io/en/stable/',
```

Comment:
I thought we wanted to have `latest` as the default

Reply:
we shall refer stable version not develop
cc: @aribornstein

Reply:
For people looking for the docs as reference I think it’s best for us to point to the stable version as latest is often under development.

Reply:
Is there a reason you prefer latest to stable?

Reply:
I would use stable too, as the stable is the version on pypi so there is less confusion between the installed and documented version